notebooks/tutorials/1_getting_started_lesson2/notebook.ipynb

```diff
@@ -0,0 +1,173 @@
yitgczYNMpaKkMmcdLzP8mQGPb9UqOTGRaXyanG812+M6mJ8jcm3M59k+/C6/Oy7xLaWXRY\nUkfO8EBowPPL7X/g5f1vhAQ9Xp+X1w+vZvmHT/XZ8vl043miNQYWj74BjUrTY42fz+ej0WYKWFJL\nRGkMDIvPorS5PCAXjLT4ZdDoKW+t5tWDK3n35EY+6mbculbZXX6IH3/6a3Ze3Nfrfcpbqnlqy+9C\natn7yrmmUh756Gn+eeqTPu/r9rpZWfAeRzvNe68GBkXA89gnv+L1w6vD/k3K8MRqRRvECf6AZ3/l\nUeosjUxIG8Ps7Om0Oc2cN5V22V+aeF2fOxuA8Smj0Km13XrKh8Pj87KjdC8GjZ7r88T3ys+aglKh\n5EhQdkkQhEAfnp6Q6niGSgNSu9sReGicabqA0+MkJSqJVkc7rx9ePWSle9ciTq/4PWs71apFkrVJ\nE/G+ZlWHGgeDHlIHKgu6rLAFZ3hArIuU6nBuHblAzPKU7AqMi52RGika1HpqzPVDNnt8ubnQLN6v\nc7KnMy5F7CkV7nkSjNvrxuPzEKWNCrwm1YE02oamrE2avOdnTwW+XMBTVHeqz7JLm9uOTq1DFUYy\nPippODa3PUQaZvKPOylR4QMegBlZk3F53ZxuKg55XZImpkYnsTBvNkqFsk8GSSDeI1VttShQhEjD\ngunci6fdaeG8qYxD1UVsK90NiPOP1w7+nS8uHsDpcfJFee+Po8lqotFqYkLaWPRqHRNSRlPZVkNr\nN01Xm+2teH3esHVNY5NH4PK6qfTL1TqaUHcEoQqFgrz4bGrNDYHs2YGqruNdTwiCwMGqwiHZzNfi\ntLLq2DoATjaEL8kIx8mGc1S0VvPyvjf4rPjzPn9uq72NV/a/gdvrZkvJrkANVm/ZXrqXzSVf8MqB\nN6+6XkuDIuAx2prZW3kksKISTCDDoxMDnkRDPOkxqZT7f2yT08YyN0fsBRBO1ma0tRCvj+PBad8g\nLyGH20YtBLqvNwjH0ZrjtDjauGn4vEDdQqwuhgmpo7nQXB6Y0Hm7KZLsTGZMGtEqA2eaLlz1k8Dd\n5Yd4avNzEdPSgiDw1Jbn+M9dryEIAoW14iD3/dkPMTV9AoV1pyi8wm44Mv2Hw+NEp9KiVIQOMWNT\npIWE8AGP2Wnps1a9uq2O72xYwbG6M1/iiK8ODlYVolQoyc+agsne0uU6dQ54gtGoNHx13G24vG6+\nuHgg7Ps3+yW8N464DvBby8r0SGVrDVEaA8lRiaREJZFoiKfY2FWaGYxVajoakuERC+aHqqxNykKM\nTBxGanQyVZcYUDs8Tl7c87+8euDNPu1nczu6yNkkRiWKdTxlQbK2ZinDE0HSBjBDquOpDa3jkeYj\nqVHJJBjimZYxkdKWil4HeauL1vFvH/2UC83lZMSkdlkckkiNTkalVAWubbBE/u1jH3DGXMqvdvw3\nB6oKGJ8yCoNG36fgQZKzTfJnLqdmiG0ETnST5akPGBZ0leGNTQltMi0tfuk7GS7k+mVtqVFJzMqe\nhsnW0uv5lkRB7Ule3v8G757Y2Kf9BgNrT3wYmN+W9OG6SM9SjUrN34veZ9P57b3e1+vz8scDb9Li\naCM7NgOLy8rBqsJe7293O1h/5lM0SjVur5tXDvwtEPBeDQyKgAfECfO20j1dXg9keHQdja4mpHQ0\nNpycNo7JaeOI0hg4XH0s5AElCALNthZSDOJD7A+3P8v8XNGwYETCMFRKFacbi3sVbGwp2QXA7aNv\nCHl9TrYYbEnpvQ4bzJ4DHoVCwTBDBq2O9itasPhlEQSBD89uoaKthu2le8Nu0+4002g1cd5YytHa\nExTWnUKv1jEpdSwPTbsbgB0R9pUZfDg9rhAJg8TIxDyUCmXXDI+tQ2rV14feOWMJVpeNHWWD+/5p\nspooaS5ncto4bh99IwD7q0IlMpFsdyVuGn4dGpWGHaV7w45b0sNw0aiFqJQqDlQWXPWLKQONy+Oi\nztJIXkI2CoUChULBuJRRtDraabRGDlysbrEgPjjDIzlaDVXjgo4C/HSGxWfR5mi/JLlyeUs1XsFH\nrbmhSw1sd9jcdqLU4V3hRiUNB6AkKOAx2VtQKBQk+CV44RiXMgqDWk9R3amQ30qTVZS7S0HsTf5F\nhF29yK60OtrZVLwDlVLFV8feylMLH424rUqpIiMmlVpzA4IgUOO/HtMyJuL0uvi44XNKmstZkDuL\nn9/4GLP7EDwIgsDn/sWRSWnjAJiSPgGgW1mbdN93lrQBXdQx4UwLAK7LmUleQg4rFnyfRaOuB8Ss\ndl+Q5lV7K4/g8Xr6tO/VTImpnO1lexkWl8m0jIk02Zq7zbgFI43xv7jxxyTq41lzfAPnjd1noyWK\n6k5xpukCc7Kn88wNP0SBgm0lu3t93JvOb6fdaeHuiXewZMzN1LTX89Otv+eF3X/mU7/py0DS54Cn\nuLiYRYsW8Y9//AOA+vp6li1bxtKlS3niiSdwu8X018aNG7nnnnu4//77WbdOTMt5PB6efPJJHnzw\nQZYtW0Z1dd8cOnaW7etSkNfeKcMDHXU8MdpochOyUavU5GdNwWhrpiyot06704zb5yE5TDpbq9aS\nnzWFyraaHgupjbZmTjcWMyltLFlxGSF/m53jr+OpFn+Ybn/A05saHoBhhkwAzjSGptMrWqsD2tiB\nprKtJiCN2V62N2ytVLAV68rC92iwNDE1YwJqlZoRicMYlZhHQd3JkImvzOBFDHi62prq1FryErIp\na6kM/JYFQaDZ3hpY4eytjFTC6F+lPVZ/ps/p96sBj89Lq72NXf5x5rphM5mcPp5YbTQHqwpDfk/2\nTjU8nYnRRTNv2EzqLI1hnSlbHG0oFAqyYtOZnT2NiraagJNkm6Od/XIA1IXq9joEQWBYfFbgtXHJ\nfllbN/eq1IMnOMOTHi2uiA9Va+o6S4fjWK7/el1Kl
```

Comment:
I'd specifically bring up that looking at a plot is not enough. It might seem interesting but actually be a spurious relationship. To encourage actually using statistics to test for stuff, I would have another cell that calculates the contemporanous (and maybe also the lagged) correlation to try to back up the claim about matching directions.

For the spike, I would pull that week for each factor (`returns`, `msg_volume`, and `sentiment`) just to get a more granular view and make sure they're within a couple days of each other.